src/template.ts

```diff
@@ -110,57 +127,71 @@ function getCss(theme: string, fontSize: string) {
         font-style: normal;
         font-weight: 500;
         color: ${foreground};
-        line-height: 1.1;
+        line-height: 1.25;
         margin: 0;
     }`
 }
 
 export function getHtml(parsedReq: ParsedRequest, course: any) {
-    const { theme, md, fontSize, widths, heights } = parsedReq
-    const { square_cover_large_url, title, instructor } = course
-    const images = [square_cover_large_url]
-    const text = title
-
-    return `<!DOCTYPE html>
+  const { theme, md, fontSize, widths, heights } = parsedReq
+  const { square_cover_large_url, title, instructor } = course
+  const images = [square_cover_large_url]
+  const text = title
+  const adjustedFontSize =
+    text.length > 60 ? (text.length > 80 ? "52px" : "56px") : fontSize
+
+  return `<!DOCTYPE html>
 <html>
     <meta charset="utf-8">
     <title>Generated Image</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <style>
-        ${getCss(theme, fontSize)}
+        ${getCss(theme, adjustedFontSize)}
     </style>
     <body>
-        <div>
-            <div class="wrapper">
-                <div class="logo-holder">
-                    ${images
-            .map(
-                (img, i) =>
-                    getPlusSign(i) + getImage(img, widths[i], heights[i])
-            )
-            .join("")}
-                </div>
-                <div class="info-holder">
-                    <div class="heading">${emojify(
-                md ? marked(text) : sanitizeHtml(text)
-            )}</div>
-                    <div class="divider"></div>
-                    <div class="with-author-holder">
-                        <div>with</div>
-                        <div class="author-name">${emojify(
-                md ? marked(instructor.full_name) : sanitizeHtml(instructor.full_name)
-            )}</div>
-                    </div>
+        <div class="wrapper">
+            ${getImage(
+              "https://d2ddoduugvun08.cloudfront.net/items/0s0k2J0j3i3O2m0D1k2K/eggo.svg",
```

Comment:
hardcoded link to eggo for now, need to be changed to the one hosted on our side (I failed to find the way to use image placed locally in the project)